core-bundle/src/Controller/BackendTemplateStudioController.php

```diff
@@ -127,4 +149,17 @@ public function __construct(
             'tree' => $prefixTree,
         ]);
     }
+
+    private function getTemplateNameInformation(string $logicalName): array
+    {
+        [$namespace, $shortName] = ContaoTwigUtil::parseContaoName($logicalName);
+
+        return [
+            'name' => $logicalName,
+            'short_name' => $shortName ?? '?',
+            'namespace' => $namespace ?? '?',
+            'identifier' => ContaoTwigUtil::getIdentifier($shortName) ?? '?',
+            'extension' => ContaoTwigUtil::getExtension($shortName) ?? '?',
```

Comment:
Did you mean this or can the `?? '?'` part be removed completely?

```suggestion
            'identifier' => ContaoTwigUtil::getIdentifier($shortName) ?: '?',
            'extension' => ContaoTwigUtil::getExtension($shortName) ?: '?',
```

![](https://github.com/user-attachments/assets/1cf0a48a-6894-4651-8c34-5818496a6147)